pontoon/base/static/js/tour.js

```diff
@@ -45,7 +63,16 @@ $(function () {
         text:
           "Pontoon is a web-based, What-You-See-Is-What-You-Get (WYSIWYG), " +
           "localization (l10n) tool. At Mozilla, we currently use Pontoon " +
-          "to localize various Mozilla project."
+          "to localize various Mozilla project.",
+        listeners: {
+          beforeStep: function() {
+            if(tourStatus != 0)
+              Sideshow.gotoStep(tourStatus+1);
+          },
+          afterStep: function() {
+            update_tour_status(1);
```

Comment:
I don't like that you have to set the step numbers manually. If we ever add a step near the beginning, we'll have to change all the following numbers, and that's going to be very annoying and hard to maintain. Can you find a different solution so that the numbers aren't hard-coded?